src/templates/base/base.html

```diff
@@ -482,12 +482,17 @@
           element.hidden = false;
         });
 
-        // Language and Year switching
+        // Language, Year and ToC switching
         var languageYearSwitchers = document.querySelectorAll('.language-switcher select, .year-switcher select, .table-of-contents-switcher select');
         for (var i = 0; i < languageYearSwitchers.length; i++) {
           languageYearSwitchers[i].addEventListener('change', function(e) {
+
             //Reset the selector back in case user uses Back button
             var selectedValue = e.target.value;
+
+            if (selectedValue.startsWith('/static/pdfs')) {
+              gtag('event', 'ebook-click', { 'event_category': 'clicks', 'event_label': 'mobile menu ' + document.location.pathname, 'transport_type': 'beacon', 'value': 1 })
+            }
```

Comment:
Any concerns about brittleness? This statement sniffs out if the option appears to be the ebook download. Why not have the ebook download option unambiguously say what it is, for example `data-type="ebook"`, or more generally support trackable links using something like `data-event="ebook-click"`?
```suggestion
            if (e.target.dataset.event) {
              gtag('event', e.target.dataset.event, { 'event_category': 'clicks', 'event_label': 'mobile menu ' + document.location.pathname, 'transport_type': 'beacon', 'value': 1 })
            }
```